nrf_modem/doc/fault_handling.rst

```diff
@@ -0,0 +1,48 @@
+.. __fault_handling:
+
+Modem fault handling
+####################
+
+The application core and the modem core are separate cores on nRF9160.
+When the modem core on the nRF9160 crashes, it sends a fault signal to the application core.
+
+Detecting a fault
+*****************
+
+The Modem library implements a fault handling mechanism allowing the application to be notified when the modem core has crashed.
+The application can set a modem fault handler function during initialization using the Modem library initialization parameters.
+For a complete list of modem fault reasons, see :ref:`nrf_modem_fault`.
```

Comment:
Where is this linking to? Was it supposed to link to a section in API?

Reply:
it's in `nrf_modem.h`

Reply:
Oh, I am not sure if we can directly link to it, unless its pulled into the API.rst file.

Reply:
Ok, I'll add it to `api.rst`